docs/content/en/docs/packages/cluster-autoscaler/addclauto.md

```diff
@@ -120,47 +104,12 @@ eksctl anywhere apply package -f cluster-autoscaler.yaml
 
 Cluster Autoscaler will automatically be upgraded when a new bundle is activated.
 
+It is possible to change the autoscaling configuration of a worker node group by updating the `WorkerNodeGroup.autoscalingConfiguration` either via the Full Lifecycle Controller or running an upgrade through the CLI.
+
 ## Uninstall
 
 To uninstall Cluster Autoscaler, simply delete the package
 
 ```bash
 eksctl anywhere delete package --cluster <cluster-name> cluster-autoscaler
```

Comment:
(Eventually) we should have examples here that show eksctl CLI and kubectl but don't have to get that into this PR